webapi/src/it/scala/org/knora/webapi/responders/admin/GroupsResponderADMSpec.scala

```diff
@@ -62,7 +57,7 @@ class GroupsResponderADMSpec extends CoreSpec with ImplicitSender {
           groupIri = "http://rdfh.ch/groups/notexisting"
         )
 
-        expectMsgPF(timeout) { case msg: akka.actor.Status.Failure =>
+        expectMsgPF(timeout) { case msg: Failure =>
           msg.cause.isInstanceOf[NotFoundException] should ===(true)
         }
```

Comment:
`should ===(true)` is a horrible construct. Is there a nicer way to express this? (maybe something like `assertTrue(msg.cause.isInstanceOf[...])`?) I know, this isn't the point of your PR, just cosmetics...